app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -176,12 +176,20 @@ export default Component.extend(LoadMore, {
 
     onClick("button.bulk-select-all", function () {
       this.updateAutoAddTopicsToBulkSelect(true);
-      $("input.bulk-select:not(:checked)").click();
+      for (const unchecked of [
+        ...document.querySelectorAll("input.bulk-select:not(:checked)"),
+      ]) {
+        unchecked.click();
+      }
```

Comment:
Codegolf 😉 (non-prettierfied) but either way is fine 😛 

```suggestion
      document.querySelectorAll("input.bulk-select:not(:checked)").forEach((el) => el.click());
```